rest-api/src/test/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1Test.java

```diff
@@ -95,6 +95,28 @@ void testMarkFileAsProcessedHttpStatusOk() {
 		assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
 	}
 
+	@Test
+	void testMarkFileAsUnprocessedReturnsSuccess() {
+		when(cpcFileService.unprocessFileById(anyString())).thenReturn("success!");
+
+		ResponseEntity<String> response = cpcFileControllerV1.markFileUnprocessed("meep");
+
+		verify(cpcFileService, times(1)).unprocessFileById("meep");
+
+		assertThat(response.getBody()).isEqualTo("success!");
+	}
+
+	@Test
+	void testMarkFileAsUnprocessedHttpStatusOk() {
```

Comment:
We should write another test to test that the feature flag correctly disables the unprocessed API.